src/WebViewer.svelte

```diff
@@ -1,6 +1,7 @@
 <script>
 
-	import { onMount, createEventDispatcher } from 'svelte';
+  import { onMount, createEventDispatcher } from 'svelte';
+  import WebViewer from '@pdftron/webviewer';
```

Comment:
I dont think we're acutally using this because we're still calling window.WebViewer below